napture/src/b9/html.rs

```diff
@@ -66,7 +67,7 @@ async fn parse_html(mut url: String) -> Result<(Node, Node)> {
         Some(head) => head,
         None => {
             return Err(html_parser::Error::Parsing(
-                "Couldn't find head. Invalid HTML?".to_owned(),
+                "Couetch_imageldn't find head. Invalid HTML?".to_owned(),
```

Comment:
What happened to this ?

Reply:
Accidentally triggered a paste here, idk. Will fix.